migrator/migrations/m_183_to_m_184_create_declarative_config_health_table/migration_test.go

```diff
@@ -0,0 +1,59 @@
+//go:build sql_integration
+
+package m175tom176
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/uuid"
```

Comment:
Wrong UUID package, we should use the stackrox one (this also leads to the go.mod change which is unwanted).